DYDX-Bots/Parameter_Changes_Monitor/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "dydx-17",
+  "version": "0.0.1",
+  "description": "Monitor parameter changes",
```

```suggestion
  "description": "Monitor parameter changes in dYdX Safety and Liquidity Modules",
```
a suggestion to have a more detailed description.